src/ape/managers/converters.py

```diff
@@ -246,6 +246,8 @@ def _converters(self) -> Dict[Type, List[ConverterAPI]]:
             Decimal: [],
             list: [ListTupleConverter()],
             tuple: [ListTupleConverter()],
+            bool: [],
+            str: [],
```

Comment:
Also had to add these so that we could convert these types without issue. There's no converters I can think of for these types